test/e2e/monitoring/alerts_test.go

```diff
@@ -54,6 +116,11 @@ var _ = Context("Prometheus Alerts", func() {
 			}
 		})
 
+		It("configuration: role-binding should point to the prometheus serviceAccount", func() {
```

Comment:
The reason this test is added is because I had times where I ran the test manually and forgot to set the `MONITORING_NAMESPACE` env var + reset the CNAO manifests. This results in it being defaulted to the d/s value, and the tests to fail for "unexplained" reason.